### PR TITLE
B 21428 add market code to shipments int

### PR DESCRIPTION
**Pull request by cameroncaci**
### Summary

Preps downstreams by adding an optional market code attribute to mto shipment model and table

### CODE REVIEW

**Review by traskowskycaci — COMMENTED** [review]
Any swagger updates we need to make for this?

File: migrations/app/schema/20241001193619_market-code-enum.up.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- create and set market code
+create type market_code_enum as enum ('i', 'd');
+ALTER TABLE mto_shipments ADD COLUMN market_code market_code_enum;
+COMMENT ON COLUMN "mto_shipments"."market_code" IS 'Market code indicator for the shipment. i for international and d for destination.';
```
Comment (danieljordan-caci) [inline migrations/app/schema/20241001193619_market-code-enum.up.sql:4]: To keep up with how we've been doing previous migrations, could you remove the quotes from this comment line?
```
COMMENT ON COLUMN mto_shipments.market_code IS
```

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

**Review by begrohmann — APPROVED** [review]

File: migrations/app/schema/20241001193619_market-code-enum.up.sql (hunk 1 above)
Comment (cameroncaci) [reply migrations/app/schema/20241001193619_market-code-enum.up.sql]: Resolved via https://github.com/transcom/mymove/pull/13818/commits/4455101e5a99f77b20d99f0c95aa69fa10f99d2d

**Comment by cameroncaci** [comment]
> Any swagger updates we need to make for this?

@traskowskycaci 
I don't think it'd be good to have swagger return data that is only possible to be null - could send mixed messages to third parties. I'd wager that'd be better for the BLI which will enhance the Prime API

**Review by traskowskycaci — APPROVED** [review]
Looks good! Failing server tests are just flaky

File: migrations/app/schema/20241001193619_market-code-enum.up.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+-- create and set market code
+create type market_code_enum as enum ('i', 'd');
+ALTER TABLE mto_shipments ADD COLUMN market_code market_code_enum;
```
Comment (danieljordan-caci) [inline migrations/app/schema/20241001193619_market-code-enum.up.sql:3]: `ADD COLUMN IF NOT EXISTS` here plz

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]

**Review by danieljordan-caci — APPROVED** [review]
vv happy thank u

Screenshot 2024-10-07 at 9 46 51 AM

**Pull request merged**